### ISSUE TITLE
Axios works on iOs but not on Android - Request failed with status code null

### ISSUE THREAD

**Issue by jechazelle** [NativeScript/NativeScript#8557]
Hello,
when I execute an Axios request on Android, I have this response :
`Request failed with status code null`

The request works on iOs.

**Environment**
- CLI: 6.5.0
- tns-android: 6.5.0
- tns-ios: 6.5.0
- Axios : 0.19.2

**Describe the bug**
When I execute a request on iOs, I have a response, but when I execute a request on Android, I have this message :
`Request failed with status code null`

**To Reproduce**
```
import axios from 'axios';

return axios.post(`http://127.0.0.1:8000/api/login`, {
        email   : user.email,
        password: user.password
    })
    .then((response) => {
        console.log('response : ', response)
        return response;
    })
    .catch((error) => {
        // return : Request failed with status code null
        console.log(error)
        return error.response;
    })

```

I tried to add this in AndroidManifest.xml, but same error :
`android:usesCleartextTraffic="true"`

And I tried with this, but same result : 
```
return axios.post(`http://127.0.0.1:8000/api/login`, {
        email   : user.email,
        password: user.password
    }, {
    headers: {
        'Accept': 'application/json, text/plain, */*',
        'Content-Type': 'application/json'
}})
```

I read this issue [https://github.com/NativeScript/NativeScript/issues/7739](https://github.com/NativeScript/NativeScript/issues/7739) but I don't understand if it's the same problem...

**Comment by jechazelle** [comment]
I fixed this error : I used 10.0.2.2 to localhost access (127.0.0.1)

More informations here : 
 - [https://developer.android.com/studio/run/emulator-networking.html](https://developer.android.com/studio/run/emulator-networking.html)

- [https://stackoverflow.com/questions/5806220/how-to-connect-to-my-http-localhost-web-server-from-android-emulator-in-eclips](https://stackoverflow.com/questions/5806220/how-to-connect-to-my-http-localhost-web-server-from-android-emulator-in-eclips)

**Comment by klachhab** [comment]
If your are using HTTP add this line 'android:usesCleartextTraffic="true"' to the application tag inside the manifest file
